StreamChatClient_v3/Models/Message.swift

```diff
@@ -8,9 +8,23 @@ import Foundation
 public typealias MessageId = String
 
 public struct MessageModel<ExtraData: ExtraDataTypes> {
-    let id: MessageId
+    public let id: String
     public let text: String
+    public let type: String
+    public let command: String?
+    public let createdDate: Date
+    public let updatedDate: Date
+    public let deletedDate: Date?
```

Comment:
Not sure what's our current policy about naming `Date` variables 🤔

Reply:
I've left the naming to another PR, alongside ChannelListController

Reply:
I do the same, we should make a separate PR to cleanup at once.